templates/electron-app/electron-entry.js

```diff
@@ -0,0 +1,65 @@
+const electron = require('electron')
```

Comment:
in grommet we follow the convention to add a semi-colon at the end of each line. Do you mind following that?